pkg/storaged/utils.js

```diff
@@ -771,8 +771,29 @@ export function find_children_for_mount_point(client, mount_point, self) {
     return children;
 }
 
-export function get_fstab_config_with_client(client, block, also_child_config) {
-    let config = block.Configuration.find(c => c[0] == "fstab");
+export function get_fstab_config_with_client(client, block, also_child_config, subvol) {
+    function match(c) {
+        // HACK: get this from `btrfs get-default <path>`
+        // https://github.com/storaged-project/udisks/pull/1222
+        const default_subvol = "/"; // XXX - get this from btrfs
+
+        if (c[0] != "fstab")
+            return false;
+        if (subvol !== undefined) {
+            const opts = decode_filename(c[1].opts.v).split(",");
+            if (opts.indexOf("subvolid=" + subvol.id) >= 0)
+                return true;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19690-20231205-110900-7b3be2c9-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)